composer.json

```diff
@@ -21,7 +21,8 @@
         "phpactor/test-utils": "^1.0.2",
         "phpstan/phpstan": "~0.12.0",
         "phpunit/phpunit": "^9.0",
-        "phpspec/prophecy-phpunit": "^2.0"
+        "phpspec/prophecy-phpunit": "^2.0",
+        "symfony/var-dumper": "^5.2"
```

Comment:
I get your point, but so is PHPUnit. And, you could have a versioning nightmare on your hands with Symfony projects.

Reply:
I was thinking about adding this to all phpactor packages in `required-dev`, prepending it is a good solution though - although it does leave open the potential for conflicts.

Reply:
I have multiple Symfony projects in different versions, please guys.
I wonder if it could be `composer install`-ed without being in composer.json?

Reply:
https://github.com/franzliedke/studio was doing this, but I don't know if they updated to composer 2

I'm well aware of the issues that might bring a global version, I deal with it every day :)
My goal here is not to be annoying, as painful as it is to locally deal with versioning issue it's even worse with phpactor because of the number of packages.
It could be a pain later to have to deal with Symfony dependencies since we have none yet in this package.

Since @dantleech was already thinking about adding it globally it's a non issue so you can resolve ;)
I didn't require any changes with this review, it's only there to gave feedback and have the opportunity to share.

Reply:
I welcome the feedback!

Reply:
> even worse with phpactor because of the number of packages.

that's not so problematic now, i can update all [packages in a few minutes](https://github.com/phpactor/phpactor-hub)